pkg/resource/sensitive.go

```diff
@@ -114,9 +115,18 @@ func GetSensitiveAttributes(from map[string]interface{}, mapping map[string]stri
 		}
 
 		for _, fp := range fieldPaths {
-			v, err := paved.GetString(fp)
+			v, err := paved.GetValue(fp)
 			if err != nil {
```

Comment:
We probably need to ignore `NotFound` error here https://github.com/crossplane/crossplane-runtime/blob/47bff13/pkg/fieldpath/paved.go#L39

Reply:
In fact, you may continue to use `GetString` if we ignore the error and check against `""` in the next step, hence remove type casting.

Reply:
@muvaf we have two different cases here:

1. No value at that path (i.e. path does not exist)
2. Path exists but set as `null`/`nil`

First case already covered with https://github.com/crossplane-contrib/terrajet/pull/149 and `NotFound` error is consumed by [expandWildcard](https://github.com/crossplane/crossplane-runtime/blob/d43d510ec5786c455983dede48878b5231de50a8/pkg/fieldpath/paved.go#L191).

This PR is fixing second case where path exists but we are getting a null value.

Reply:
Got it, I didn't realize we have `null`s in tfstate. Thanks!